src/renderers/webgl/WebGLProgram.js

```diff
@@ -510,6 +510,8 @@ function WebGLProgram( renderer, cacheKey, parameters, bindingStates ) {
 			parameters.useFog && parameters.fog ? '#define USE_FOG' : '',
 			parameters.useFog && parameters.fogExp2 ? '#define FOG_EXP2' : '',
 
+			parameters.occlusion ? '#define USE_OCCLUSION' : '#define USE_OCCLUSION',
+
```

Comment:
Both sides are equal; the right-hand side should be `''`.

Reply:
I know :-)
That is why it's WIP and still a draft PR

Reply:
@CodyJasonBennett fixed!